assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -96,6 +96,9 @@ var ajaxCallbacks = {
           $parent.append($offDisabled);
         }
 
+        // update no of subscriptions
```

Comment:
rather than adding this comment, you could make a function to let the code describe what is going on
```
updateSubscriptionCounter()
```